python/cugraph/cugraph/__init__.py

```diff
@@ -53,6 +53,9 @@
     is_multigraph,
     is_bipartite,
     is_multipartite,
+    replicate_edgelist,
+    replicate_cudf_dataframe,
+    replicate_cudf_series,
```

Comment:
If these aren't intended to be part of the user-visible API then we shouldn't put them in this file.  ( didn't think they were supposed to be user-visible, but now I'm not sure)

Reply:
I will remove it from the user-visible API for now. If in the future we change our mind we can't always bring it back